tests/test_posixDatastore.py

```diff
@@ -1,7 +1,7 @@
 #
 # LSST Data Management System
 #
-# Copyright 2008-2018  AURA/LSST.
+# Copyright 2018  AURA/LSST.
```

Comment:
Is this the new default?

Reply:
A new file written from scratch can't have a copyright year from before it existed.